src/components/Auth/UserFormDialog/UserForm.tsx

```diff
@@ -65,23 +65,23 @@ function convertFromFormUser(formUser: AuthFormUser): AddAuthUserPayload {
   return {
     ...formUser,
     emailVerified: formUser.emailVerified.length > 0 ? true : false,
-    // match mfaPhoneInfo array members to an existing enrollment, or create a new enrollment
-    mfaInfo: formUser.mfaPhoneInfo
-      ? formUser.mfaPhoneInfo.map((mfaPhoneInfo) => {
-          const existingEnrollment = formUser.mfaInfo?.find(
-            (mfaEnrollment) =>
-              mfaEnrollment.phoneInfo === mfaPhoneInfo.phoneInfo
-          );
-          return (
-            existingEnrollment || {
-              ...mfaPhoneInfo,
-              enrolledAt: new Date().toISOString(),
-              mfaEnrollmentId:
-                'AUTH-EMULATOR-UI:' + Math.random().toString(36).substring(5),
-            }
-          );
-        })
-      : undefined,
+    mfaInfo:
```

Comment:
Please extract this into a variable. Please also remove the ternary while we're at it.

```js
let mfaInfo: AddAuthUserPayload["mfaInfo"];
if (formUser.mfaEnabled[0] === 'on') {
  mfaInfo = ...
}
```